src/core/SpriteAnimator.tsx

```diff
@@ -17,12 +17,12 @@ export type SpriteAnimatorProps = {
   onEnd?: Function
   onLoopEnd?: Function
   onFrame?: Function
-  play?: boolean
   pause?: boolean
```

Comment:
Why was `play` removed? I'm not sure I see any conflicts for it.

Reply:
@CodyJasonBennett It was never used, autoPlay does the same thing. It was a clean-up change.

Reply:
I see. We're not able to remove or rename APIs as part of a feature, but we can deprecate the property so it can be cleaned up when we release v10 of Drei.

Reply:
@CodyJasonBennett ok sure I'll make the change

Reply:
I reverted the change and marked the property as deprecated on the ReadMe